lib/db/schemaupdater.go

```diff
@@ -83,64 +83,41 @@ func (db *schemaUpdater) updateSchema() error {
 		return err
 	}
 
-	prevRepairVersion, _, err := miscDB.Int64("dbRepairVersion")
+	prevMigration, _, err := miscDB.Int64("dbMigrationVersion")
 	if err != nil {
 		return err
 	}
 
```

Comment:
`if prevMigration < prevVersion { prevMigration = prevVersion }` or something somewhere to cover legacy + impossible oddities?